src/main/resources/db/changelog/20220722000000-add_delete_cascade_to_consumer_fks.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="20220722000000-1" author="jalbrech">
+        <preConditions onFail="MARK_RAN">
+            <foreignKeyConstraintExists foreignKeyName="cp_content_tags_consumer_fk"/>
+        </preConditions>
+        <dropForeignKeyConstraint baseTableName="cp_consumer_content_tags" constraintName="cp_content_tags_consumer_fk"/>
+    </changeSet>
+
+    <changeSet id="20220722000000-2" author="jalbrech">
+        <addForeignKeyConstraint baseTableName="cp_consumer_content_tags"
+            baseColumnNames="consumer_id"
+            referencedTableName="cp_consumer"
+            referencedColumnNames="id"
+            constraintName="cp_content_tags_consumer_fk"
+            onDelete="CASCADE" />
+    </changeSet>
+
+    <changeSet id="20220722000000-3" author="jalbrech">
+        <preConditions onFail="MARK_RAN">
+            <foreignKeyConstraintExists foreignKeyName="fk278ce810390317d"/>
+        </preConditions>
+        <dropForeignKeyConstraint baseTableName="cp_consumer_facts" constraintName="fk278ce810390317d"/>
+    </changeSet>
+
+    <changeSet id="20220722000000-4" author="jalbrech">
+        <addForeignKeyConstraint baseTableName="cp_consumer_facts"
+            baseColumnNames="cp_consumer_id"
+            referencedTableName="cp_consumer"
+            referencedColumnNames="id"
+            constraintName="cp_consumer_facts_consumer_fk"
+            onDelete="CASCADE" />
+    </changeSet>
+
+    <changeSet id="20220722000000-5" author="jalbrech">
+        <preConditions onFail="MARK_RAN">
+            <foreignKeyConstraintExists foreignKeyName="fk_cnsmr_capability_cnsmr"/>
+        </preConditions>
+        <dropForeignKeyConstraint baseTableName="cp_consumer_capability" constraintName="fk_cnsmr_capability_cnsmr"/>
+    </changeSet>
+
+    <changeSet id="20220722000000-6" author="jalbrech">
+        <addForeignKeyConstraint baseTableName="cp_consumer_capability"
+            baseColumnNames="consumer_id"
+            referencedTableName="cp_consumer"
+            referencedColumnNames="id"
+            constraintName="fk_cnsmr_capability_cnsmr"
```

Comment:
Let's update this constraint name to match the others. The other FK names beyond this point are also in a different general format (fk_... vs table_fk). We've not been consistent on naming thus far on a global basis, so it's probably not exactly critical which one we go with, but we should be consistent within a single changeset file.

Reply:
Good catch, I missed that they were using different formats! I updated them to fk_<base table name>_<reference table name>